findContact.js

```diff
@@ -0,0 +1,21 @@
+// 13. Write the findContact function that returns a string representing the contact with a given name.
+// If a contact with a given name does not exist, return undefined.
+
+function addContact(object, name, phoneNumber) {
+    return {
+        John: {
+            name: name,
+            phoneNumber: phoneNumber
+        }
+    }
+}
+
+function findContact(object, name) {
```

Comment:
`object` is not a good argument name if we have more information about the data that it contains